builds/e2e/e2e.yaml

```diff
@@ -272,4 +272,71 @@ jobs:
     - template: templates/e2e-clear-docker-cached-images.yaml
     - template: templates/e2e-run.yaml
       parameters:
-        test_type: http_proxy
\ No newline at end of file
+        test_type: http_proxy
+
+################################################################################
+  - job: mariner1_amd64
+################################################################################
+    displayName: Mariner 1.0 amd64
+    condition: eq(variables['run.EFLOW.amd64'], 'true')
```

Comment:
Where is this variable set, and what is it used for? I can't find any other references besides this job conditions...

Reply:
run.EFLOW.amd64 is defined in the ADO pipeline. This was required for the EFLOW tests due to the flaky test agents we had previously.

Reply:
Ok. I see the variable added to the long haul pipeline, but it's not defined in the end-to-end pipeline. Do we need to add it to `e2e.yaml`, especially for jobs running on 1ES agents?

Reply:
I do not think we will need to add that value, will remove it for E2E tests on mariner. Do we also want to add 1es agents for the Mariner 1.0 packages as well?

Reply:
> I do not think we will need to add that value, will remove it for E2E tests on mariner.
`run.EFLOW.amd64` is not defined in the end-to-end test pipeline, so I assume you haven't run this job? If nobody's using it and it's EOL soon, should we just remove this job?

Reply:
I have run this job, can link to the builds if you are interested. I added it to the runs directly rather than adding it to the existing pipeline variables, I planned on adding it after the PR was merged.